src/configure-apollo-network.js

```diff
@@ -65,11 +64,9 @@ const configureApolloNetwork = (context) => {
     };
 
     Object.assign(context, {
-        // Specifically we need to use the server-side Node.js versions of
-        // apollo-client and react-apollo (the ones we use on the main site
-        // don't include the server-side rendering logic).
+        // Specifically we need to use the server-side Node.js version of
+        // apollo-client).
         ApolloClient: ApolloClient,
-        ReactApollo: ReactApollo,
```

Comment:
No more `react`-specific imports in the server. The client code being rendered is now responsible for these. The webpacked test cases test this out with code built in webapp using the current webpack build.